Source/KakapoServer.swift

```diff
@@ -91,11 +103,15 @@ public final class KakapoServer: NSURLProtocol {
     
     /// Start loading the matched requested, the route handler will be called and the returned object will be serialized.
     override public func startLoading() {
-        KakapoServer.routers.filter { $0.canInitWithRequest(request) }.first!.startLoading(self)
+        if let routerIndex = KakapoServer.routers.indexOf({ $0.canInitWithRequest(request) }) {
+            KakapoServer.routers[routerIndex].startLoading(self)
+        }
     }
     
-    /// Not implemented yet, does nothing ATM. https://github.com/devlucky/Kakapo/issues/88
+    /// Stops the loading of the matched request.
     override public func stopLoading() {
-        /* TODO: implement stopLoading for delayed requests https://github.com/devlucky/Kakapo/issues/88 */
+        if let routerIndex = KakapoServer.routers.indexOf({ $0.canInitWithRequest(request) }) {
+            KakapoServer.routers[routerIndex].stopLoading(self)
+        }
```

Comment:
As mentioned, can you try to just keep a private boolean flag that, when set, will prevent the Server to forward the call to the routers in `startLoading()`?

1. `stopLoading` -> set flag to true
2. `startLoading` -> check if flag is set; if it's the case, we don't even forward calls to router, just do nothing.